### PR TITLE
Add basic engine output selector tabs.

### PR DESCRIPTION
**Pull request by AlexOVeritone**

### CODE REVIEW

File: packages/veritone-widgets/src/redux/modules/mediaDetails/saga.js [hunk 1]
```diff
@@ -149,7 +152,7 @@ function* loadEngineCategoriesSaga(widgetId, mediaId, callback = noop) {
         return;
       }
       if (category.engines.some(engine => engine.status !== 'complete')) {
-        category.status = 'in_progress';
+        category.status = 'inprogres';
```
Comment (ChrisPelletier) [inline packages/veritone-widgets/src/redux/modules/mediaDetails/saga.js]: is this a typo?

File: packages/veritone-react-common/src/components/MediaDetails/index.js [hunk 2]
```diff
@@ -12,13 +13,23 @@ export default class MediaDetails extends Component {
   };
 
   state = {
-    selectedEngineCategoryId: ''
+    selectedEngineCategory: this.props.engineCategories && this.props.engineCategories.length ? this.props.engineCategories[0] : null
```
Comment (ChrisPelletier) [inline packages/veritone-react-common/src/components/MediaDetails/index.js]: Is this something we can move to redux?

File: packages/veritone-widgets/src/redux/modules/mediaDetails/saga.js (hunk 1 above)
Comment (AlexOVeritone) [reply packages/veritone-widgets/src/redux/modules/mediaDetails/saga.js]: I think I saw tried status without _ in some documentation, however, couldn't find that documentation.

File: packages/veritone-react-common/src/components/MediaDetails/index.js (hunk 2 above)
Comment (AlexOVeritone) [reply packages/veritone-react-common/src/components/MediaDetails/index.js]: yes, and it would feed it down the stack I think

**Pull request merged**